onnx/cpp2py_export.cc

```diff
@@ -146,6 +147,38 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
     return OpSchemaRegistry::get_all_schemas_with_history();
   });
 
+  defs.def(
+      "get_all_functions",
+      [](const std::string& domain)
+          -> std::unordered_map<std::string, std::vector<py::bytes>> {
+        std::multimap<std::string, std::unique_ptr<FunctionProto>> temp_ptr_map;
+        std::unordered_map<std::string, std::vector<py::bytes>> temp_map;
+        FunctionBuilderRegistry& function_registry =
+            FunctionBuilderRegistry::OnnxInstance();
+
+        Common::Status status =
+            function_registry.GetFunctions(domain, &temp_ptr_map);
+        for (auto iter = temp_ptr_map.begin(); iter != temp_ptr_map.end();
+             ++iter) {
+          std::string bytes;
+          if (!iter->second->SerializeToString(&bytes)) {
+            throw std::runtime_error(
+                "Failed to serilize registered function for '" + iter->first +
+                "'!");
+          }
+          if (!temp_map.count(iter->first)) {
```

Comment:
you can use `[]` of unordered_map to implicitly create an entry if the key doesn't already exist.